client/components/popover/test/index.test.js

```diff
@@ -1,4 +1,4 @@
-import { render, screen, act } from '@testing-library/react';
```

Comment:
nit:  like the `createRoot` change, this is a code change driven by a dependency update rather than a pure linting fix.

Reply:
Reverted it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4606/commits/2a0581fee1b2953b68e97462c12df20447f76acf